src/vni-manager.js

```diff
@@ -39,6 +39,20 @@ module.exports = function( node ) {
     _.extend( node, 
         {
 
+            /**
+             * Calls fn for each known vni in the system
+             *
+             * @this node instance
+             * @return node instance node for easy chaining
+             */
+            forEachVni: function(fn, thisArg) {
```

Comment:
For my own edification, why not let the caller (framework-ondata) iterate through the vnids, using _.each, instead of writing a custom forEachVni, which basically does the same thing as _.each?